spec/lib/hyrax/hirmeos/metrics_tracker_spec.rb

```diff
@@ -4,14 +4,39 @@
 
 RSpec.describe Hyrax::Hirmeos::MetricsTracker do
   subject(:tracker) { described_class.new }
-  let(:work) { create(:work) }
+  let(:work) { create(:work_with_files) }
   let(:file_set) { create(:file_with_work) }
+  let(:hirmeos_uuid) { "48b61e0a-f92c-4533-8270-b4caa98cbcfb" }
+  let(:hirmeos_work_data) do
+    {
+      count: 14,
+      code: 200,
+      data: [
+        {
+          URI: "https://dashboard.bertie.ubiquityrepo-ah.website/concern/pacific_images/92cfa1e1-4ab3-4fd3-9140-90eda7cd5ed9",
+          canonical: true,
+          score: 0,
+          URI_parts: {
+            scheme: "https",
+            value: "dashboard.bertie.ubiquityrepo-ah.website/concern/pacific_images/92cfa1e1-4ab3-4fd3-9140-90eda7cd5ed9"
+          },
+          work: {
+            URI: [],
+            type: "repository-work",
+            title: ["Pictures of TWICE"],
+            UUID: hirmeos_uuid
+          }
+        }
+      ]
+    }
+  end
 
   before do
     Hyrax::Hirmeos::MetricsTracker.username = "UsernameTest"
     Hyrax::Hirmeos::MetricsTracker.password = "Password"
-    body = "{\"count\": 14, \"code\": 200, \"data\": [{\"URI\": \"https://dashboard.bertie.ubiquityrepo-ah.website/concern/pacific_images/92cfa1e1-4ab3-4fd3-9140-90eda7cd5ed9\", \"canonical\": true, \"score\": 0, \"URI_parts\": {\"scheme\": \"https\", \"value\": \"dashboard.bertie.ubiquityrepo-ah.website/concern/pacific_images/92cfa1e1-4ab3-4fd3-9140-90eda7cd5ed9\"}, \"work\": {\"URI\": [], \"type\": \"repository-work\", \"title\": [\"Pictures of TWICE\"], \"UUID\": \"48b61e0a-f92c-4533-8270-b4caa98cbcfb\"}}]}" # rubocop:disable Layout/LineLength
-    stub_request(:get, Addressable::Template.new("#{Hyrax::Hirmeos::MetricsTracker.translation_base_url}/translate?uri=urn:uuid:{id}")).to_return(status: 200, body: body)
+    path = "#{Hyrax::Hirmeos::MetricsTracker.translation_base_url}/translate?uri=urn:uuid:{id}"
```

Comment:
Is this missing a `#` for the second interpolation?
```suggestion
    path = "#{Hyrax::Hirmeos::MetricsTracker.translation_base_url}/translate?uri=urn:uuid:#{id}"
```

Reply:
Seems to be a wildcard character to stub requests :) 

I didn't know about it either so, cheers @BertZZ !